pkg/runtime/deno/runtime.go

```
@@ -225,65 +164,15 @@ func (rt *Runtime) effectiveCryptoDeriver() ipc.SubKeyDeriver {
-// daemon startup. Nil disables provider task: lookups.
-func (rt *Runtime) SetProviderRunner(p envresolve.ProviderRunner) {
-	rt.providerRunner = p
-}
-
-// SetEnvResolver wires the daemon-scoped env resolver whose TTL cache
-// survives across task launches (issue #242). When set, envresolver()
-// returns it instead of constructing a fresh instance per Run.
-func (rt *Runtime) SetEnvResolver(r *envresolve.Resolver) {
-	rt.sharedResolver = r
-}
+func (rt *Runtime) effectiveTestGuard() func(taskID string) error { return rt.live().TestGuard }
```

golangci-lint (2.12.2)

[error] func (*Runtime).effectiveTestGuard is unused

(unused)